client/rest-high-level/src/test/java/org/elasticsearch/client/IndicesClientIT.java

```diff
@@ -1533,24 +1530,6 @@ public void testAnalyze() throws Exception {
         assertNotNull(detailsResponse.detail());
     }
 
-    public void testFreezeAndUnfreeze() throws IOException {
```

Comment:
@dakrone, one of consequences of removing the freeze endpoint is that many tests for unfreeze depend on it. Do you have any concerns about removing tests that exercise the unfreeze endpoint?

Reply:
Not for these tests specifically, I think it's fine to change the client tests